r/tests/testthat/test-dplyr-summarize.R

```diff
@@ -1083,11 +1121,6 @@ test_that("summarise() can handle scalars and literal values", {
     tibble(y = 1L)
   )
 
-  expect_identical(
-    record_batch(tbl) %>% summarise(y = Expression$scalar(1L)) %>% collect(),
-    tibble(y = 1L)
-  )
```

Comment:
This is the same as the one above it, yeah?

Reply:
They're all kinda the same, this one just no longer works because of how the evaluation changed, and it didn't seem worth trying to support, it's not a natural thing to do.

Reply:
Right, right about them all being _basically_ the same.

Agreed that it's not really a useful feature, but what does the error look like when it is called? My slight worry is that folks who find themselves here might be the type that need a bit more handholding to get out of compared to others?

Reply:
I don't remember TBH. But there's no legitimate reason for anyone to directly create `Expression$scalar()`, right?

Reply:
Absolutely agree there. We can add this as follow on, but my concern is if someone has become convinced that `Expression$scalar()` is needed there, if the error is one of our more generic ones or isn't clear they won't know enough to get out of that.

Reply:
Turns out this case no longer errors after my most recent simplification, but I'm still going to keep the test deleted because I don't think this is part of the API surface we need to guarantee.